01_AppCore/Records/Bases/RecordBase.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace _01_AppCore.Records.Bases
+{
+    public class RecordBase
+    {
+        public int Id { get; set; }
+        public string Guid { get; set; }
```

Comment:
Why are there 2 identifiers (id and guid)?
Also if you want to store guid-s in the database, use the Guid type instead of a string